### PR TITLE
Moving FAQ - Download Datasets

### PR DESCRIPTION
**Pull request by Melkeb**

### CODE REVIEW

**Comment by Melkeb** [comment]
Separated the FAQs to make it easier to review

File: faqs/galaxy/datasets_download_datasets.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Download datasets
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+1. Open the datasets found under history.
```
Comment (shiltemann) [inline faqs/galaxy/datasets_download_datasets.md:10]:
```suggestion
1. Click on the dataset in your history to expand it
```

File: faqs/galaxy/datasets_download_datasets.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Download datasets
```
Comment (shiltemann) [inline faqs/galaxy/datasets_download_datasets.md:2]:
```suggestion
title: Downloading datasets
```
(just for consistency with most other FAQ titles)

**Review by shiltemann — COMMENTED** [review]
Thanks @Melkeb ! Just made a few minor suggestions about the phrasing to be more consistent with language used elsewhere in the tutorials

And good idea to split into smaller PRs :+1:

**Comment by jennaj** [comment]
Looks good

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks @Melkeb! :tada:

**Pull request merged**